async-bindgen/Cargo.toml

```diff
@@ -6,7 +6,8 @@ license = "Apache-2.0"
 
 [dependencies]
 async-bindgen-derive = { path = "../async-bindgen-derive" }
-async-std = "1.10.0"
+once_cell = "1.9.0"
 static_assertions = "1.1.0"
 thiserror = "1.0.30"
+tokio = { version = "1.15.0", features = ["rt", "rt-multi-thread"] }
```

Comment:
i know that `tokio` doesn't pull in anything by default, should we still add a `default-features = false` here to make this immediately clear?